src/erasure-code/isa/CMakeLists.txt

```diff
@@ -2,61 +2,90 @@
+    ${isal_src_dir}/erasure_code/gf_3vect_dot_prod_sse.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_dot_prod_sse.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_dot_prod_sse.asm
+    ${isal_src_dir}/erasure_code/gf_6vect_dot_prod_sse.asm
+    ${isal_src_dir}/erasure_code/gf_vect_dot_prod_sse.asm
+    ${isal_src_dir}/erasure_code/gf_2vect_mad_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_mad_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_mad_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_mad_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_6vect_mad_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_vect_mad_avx2.asm
+    ${isal_src_dir}/erasure_code/ec_highlevel_func.c
+    ${isal_src_dir}/erasure_code/gf_2vect_mad_avx.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_mad_avx.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_mad_avx.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_mad_avx.asm
+    ${isal_src_dir}/erasure_code/gf_6vect_mad_avx.asm
+    ${isal_src_dir}/erasure_code/gf_vect_mad_avx.asm
+    ${isal_src_dir}/erasure_code/ec_multibinary.asm
+    ${isal_src_dir}/erasure_code/gf_2vect_mad_sse.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_mad_sse.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_mad_sse.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_mad_sse.asm
+    ${isal_src_dir}/erasure_code/gf_6vect_mad_sse.asm
+    ${isal_src_dir}/erasure_code/gf_vect_mad_sse.asm
+    ${isal_src_dir}/erasure_code/gf_2vect_dot_prod_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_dot_prod_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_dot_prod_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_dot_prod_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_6vect_dot_prod_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_vect_dot_prod_avx2.asm
+    ${isal_src_dir}/erasure_code/gf_vect_mul_avx.asm
+    ${isal_src_dir}/erasure_code/gf_2vect_dot_prod_avx.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_dot_prod_avx.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_dot_prod_avx.asm
+    ${isal_src_dir}/erasure_code/gf_5vect_dot_prod_avx.asm
+    ${isal_src_dir}/erasure_code/gf_6vect_dot_prod_avx.asm
+    ${isal_src_dir}/erasure_code/gf_vect_dot_prod_avx.asm
+    ${isal_src_dir}/erasure_code/gf_vect_mul_sse.asm
+    ${isal_src_dir}/erasure_code/gf_2vect_dot_prod_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_2vect_mad_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_dot_prod_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_3vect_mad_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_dot_prod_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_4vect_mad_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_vect_dot_prod_avx512.asm
+    ${isal_src_dir}/erasure_code/gf_vect_mad_avx512.asm
+    ErasureCodeIsa.cc
+    ErasureCodeIsaTableCache.cc
+    ErasureCodePluginIsa.cc
+    xor_op.cc
+  )
+elseif(HAVE_ARMV8_SIMD)
+  set(isa_srcs
+    ${isal_src_dir}/erasure_code/ec_base.c
+    ${isal_src_dir}/erasure_code/aarch64/ec_aarch64_highlevel_func.c
```

Comment:
```
CMake Error at src/erasure-code/isa/CMakeLists.txt:90 (add_library):
  Cannot find source file:

    /home/jenkins-build/build/workspace/ceph-pull-requests-arm64/src/isa-l/erasure_code/aarch64/ec_aarch64_highlevel_func.c
```

Reply:
It must to update isa-l(master is best,at least to 02a86df).

Reply:
could you bump the isa-l submodule in a separated commit in this changeset?

Reply:
The latest stable release is isa-l 2.28, it doesn't contains ec for aarch64. the next release 2.29 which have ec for aarch64 is coming soon on Jan 2020. Am I update to master or wait for 2.29?

Reply:
i am inclined to hold this change back until isa-l v2.29 rolls out.

Reply:
ok, I see.

Reply:
https://github.com/intel/isa-l/tree/v2.29.0


@rosinL FYI

Reply:
There are some compile errors for ceph on x86 caused by some file changes in isa-l 2.29, to update isa-l to v2.29, is the follow steps right? some advices?
1. a PR update https://github.com/ceph/isa-l to 2.29
2. fix bugs caused  by 2.29
3. update submodule isa-l and commit in this PR
@tchaikov

Reply:
@rosinL i can help on the first step after getting the write access to ceph/isa-l. we can include step 2 and 3 in this PR.

Reply:
@rosinL i just sync'ed ceph/isa-l with upstream. could you continue on with the step 2 and step 3?

Reply:
ok，we are working on it